### DISCUSSION TITLE
ffmepg not recognised by vedo

### DISCUSSION THREAD

**Discussion by jkissi** [marcomusy/vedo#701] [Q&A] [answered]
Hi Marco!

I keep getting the following error, despite the fact that I have ffmepg installed and also registered on the path in my system. 
`
[vedo.io.py:2031] ERROR: backend ffmpeg returning error: 1
`
Is there some sort of special thing I have to do to get vedo to recognise the ffmepg?

Cheers
j

**Comment by marcomusy** [comment, marked as answer]
Uhm, is it a linux system? you may try to modify manually the command
```
vid = Video()
vid.command = "ffmpeg -loglevel panic -y -r"
vid.options = "-b:v 8000k"
```

Another possibility is:
```
vid = Video(backend='cv')
```
which uses cv2

If everything fails you 'll need to save the screenshots and merge them manually with some external software i'm afraid..

**Reply by jkissi** [reply]
No it is a windows system....
